jep/200/README.adoc

```diff
@@ -123,6 +123,18 @@ When a class is (first) rejected, a warning is printed to the system log.
 This is particularly useful when scanning large automated test results for regressions caused by this change.
 Typically the rejection will also trigger an exception which gets displayed in some other manner.
 
+== Extensibility
+
+Plugin developers will be able to whitelist the classes in their libraries/plugins using
+the following approaches:
+
+0. `Jenkins-ClassFilter-Whitelisted` manifest entry (link:https://github.com/jenkinsci/lib-jenkins-maven-embedder/pull/15[example for Maven])
+0. By providing a custom `META-INF/hudson.remoting.ClassFilter` resource file in the plugin
+0. By implementing the `CustomClassFilter` extension point and implementing `permits()` methods
+
+The first two approaches do not require Jenkins core requirement update in plugins.
+Old Hudson plugins may also need additional manifest properties to be processed as Jenkins plugins by the engine.
```

Comment:
As mentioned elsewhere, I do not think this is correct.

Reply:
https://github.com/jenkinsci/jenkins/pull/3120#discussion_r158022118

I am checking old plugins, no hits so far. Likely I will just make manifest entries explicit